bool go_straight
bool turn_left
bool turn_right
bool turn_back
float32 go_x
float32 go_y